### PR TITLE
Drop navigation tabs on top

### LINKED ISSUE #28: Drop "Home" tab
Is there a way to drop the "Home" tab in the header? That introduces a lot of space that is useless unless we add more tabs (and the logo already performs the home function)

### PR DESCRIPTION
**Pull request by ocaisa**
Fixes #28 
If we only have a `Home` tab then that is pretty useless

### CODE REVIEW

**Comment by ocaisa** [comment]
Relevant documentation: https://squidfunk.github.io/mkdocs-material/getting-started/#tabs

File: mkdocs.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@ theme:
     - tabs
   logo: img/logos/EESSI.png
   favicon: img/logos/EESSI.png
+  feature:
+    tabs: false
```
Comment (boegel) [inline mkdocs.yml:14]: just remove the `- tabs` on line 10 instead?

Comment (ocaisa) [reply mkdocs.yml:14]: Yup, fixing now

**Comment by ocaisa** [comment]
Tested, works on rendered docs

**Pull request merged**